harness/determined/cli/command.py

```diff
@@ -224,8 +224,12 @@ def list_tasks(args: Namespace) -> None:
         if "workspaceId" in item:
             wId = item["workspaceId"]
             del item["workspaceId"]
-            assert wId in w_names, f"workspace id {wId} is not found"
-            item["workspaceName"] = w_names[wId]
+            # TODO: come up with a better fix that can convey workspace name?
+            assert wId == 0 or wId in w_names, f"workspace id {wId} is not found"
```

Comment:
does this mean that we expect tensorboards (or some ntsc) to report that they have a workspace id of 0?